projects/ziti-console-lib/src/lib/shared/list-page-service.class.ts

```diff
@@ -196,7 +196,7 @@ export abstract class ListPageServiceClass {
 
     public openEditForm(itemId = '', basePath?) {
         if (isEmpty(itemId)) {
-            itemId = '/create';
+            itemId = 'create';
```

Comment:
Will this break the form fields again?

Reply:
I don't think so. This `/` isn't needed because it already get's included in the route on line 202 below